rancher2/schema_cluster_aks_config.go

```diff
@@ -14,6 +14,7 @@ var (
 	clusterAKSAgentStorageProfile = []string{"ManagedDisks", "StorageAccount"}
 	clusterAKSNetworkPlugin       = []string{"azure", "kubenet"}
 	clusterAKSNetworkPolicy       = []string{"calico"}
+	clusterAKSAgentPoolType       = []string{"AvailabilitySet", "VirtualMachineScaleSets"}
```

Comment:
As just 2 values available to [`LoadBalancerSku`](https://github.com/Azure/azure-sdk-for-go/blob/master/services/containerservice/mgmt/2019-08-01/containerservice/models.go#L68) argument, make sense to add validation to it??

```
+	clusterAKSLoadBalancerSku       = []string{"basic", "standard"}
```